File: 1-iniciante/1151.py
fib=[0,1]
x=int(input())
for i in range(x):
    if(i!=x-1): print(fib[0], end=' ')
    else: print(fib[0])
    fib[1]+=fib[0]
    fib[0]=fib[1]-fib[0]

File: 1-iniciante/1151.cpp
#include <iostream>
#include <stdio.h>
using namespace std;
//
int main(){
    int fib;
    double v[2] = {0,1};
    cin>>fib;
    for(int i=0;i<fib;i++){
        printf("%.0f",v[0]);
        if(i!=fib-1){
          cout<<" ";   
        }
        v[1]=v[1]+v[0];
        v[0]=v[1]-v[0];
    } 
    cout<<"\n";
    return 0;
}